crates/punktf-cli/src/util.rs

```diff
@@ -134,10 +137,146 @@ pub fn log_deployment(deployment: &Deployment) {
 	}
 
 	if !out.is_empty() {
-		log::error!("{}", out);
+		if print {
+			println!("{}", out);
+		} else {
+			log::error!("{}", out);
+		}
+
+		out.clear();
+	}
+
+	(files_success, files_skipped, files_failed)
+}
+
+/// Logs all deployed links together with the status.
+fn log_links(out: &mut String, deployment: &Deployment, print: bool) -> (usize, usize, usize) {
```

Comment:
that's a large FN. split it up in smaller ones

Reply:
Is split into parts. Thanks!